tests/network/curl_server.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2015-2018 SUSE Linux GmbH
```

Comment:
better use the same as in config_services.pm

Reply:
And as this is a new file only `2019` is needed for the year.

Reply:
om, done! Thanks